v2f/src/main/scala/org/broadinstitute/monster/etl/v2f/V2FExtractions.scala

```diff
@@ -0,0 +1,256 @@
+package org.broadinstitute.monster.etl.v2f
+
+import java.io.InputStreamReader
+import java.nio.channels.Channels
+
+import better.files._
+import com.github.tototoshi.csv.{CSVFormat, CSVReader, TSVFormat}
+import com.spotify.scio.ScioContext
+import com.spotify.scio.values.SCollection
+import io.circe.{Json, JsonObject}
+import org.apache.beam.sdk.io.FileIO
+import org.apache.beam.sdk.io.FileIO.ReadableFile
+
+import scala.util.matching.Regex
+import util.Try
+
+/**
+  * Ingest step converting and tranforming tsv's from v2f.
+  *
+  */
+object V2FExtractions {
+
+  /**
+    * Given a root path that contains tsv(s) get the tsv as a ReadableFile
+    *
+    * @param tsvPath the root path containing tsv's to be converted
+    * @param context the pipeline context for converting
+    */
+  def getReadableFile(
+    tsvPath: String,
+    context: ScioContext
+  ): SCollection[FileIO.ReadableFile] =
+    context.wrap {
+      context.pipeline.apply(FileIO.`match`().filepattern(tsvPath))
+    }.applyTransform[ReadableFile](FileIO.readMatches())
+
+  /**
+    * Given the ReadableFile that contains tsv(s) convert each tsv to json and get is filepath
+    *
+    * @param tableName the name of the tsv table that was converted to json
+    */
+  def tsvToJson(
+    tableName: String
+  ): SCollection[FileIO.ReadableFile] => SCollection[(JsonObject, String)] =
+    _.transform(s"converting $tableName tsv(s) to json") { collection =>
```

Comment:
I'd clean these descriptions up, since they're displayed values.
```suggestion
    _.transform(s"Convert $tableName TSVs to JSON") { collection =>
```
And similar cleanups for the other labels